src/wh_client_cryptocb.c

```diff
@@ -752,21 +757,36 @@ static int _handleSha256Dma(wc_CryptoInfo* info, void* inCtx, whPacket* packet)
 
 
 #if defined(HAVE_FALCON) || defined(HAVE_DILITHIUM)
-static int _handlePqcSigKeyGen(whClientContext* ctx, wc_CryptoInfo* info)
+static int _handlePqcSigKeyGen(whClientContext* ctx, wc_CryptoInfo* info,
+                               int useDma)
 {
     int ret = CRYPTOCB_UNAVAILABLE;
 
     /* Extract info parameters */
-    WC_RNG* rng  = info->pk.pqc_sig_kg.rng;
     int     size = info->pk.pqc_sig_kg.size;
     void*   key  = info->pk.pqc_sig_kg.key;
     int     type = info->pk.pqc_sig_kg.type;
 
+#ifndef WOLFHSM_CFG_DMA
+    if (useDma) {
+        /* TODO: proper error code? */
+        return WC_HW_E;
+    }
+#endif
```

Comment:
```suggestion
#else
    (void)useDma;
#endif
```

Reply:
I don't think we need to void this here, since this #else would be the case where `WOLFHSM_CFG_DMA` is defined, and that flag will be checked in the `if` statement below. Maybe you missed the macro switch was `ifndef WOLFHSM_CFG_DMA` and not `ifdef ...`? 

The idea is that if you don't have `WOLFHSM_CFG_DMA` defined and you pass the `useDmaFlag` then the function returns error. If `WOLFHSM_CFG_DMA` is defined, then that argument will always be checked, so don't think we need the `(void)` cast on an unused arg.

Reply:
Hah!!  Yup.  I was doing the opposite logic.  No changes necessary!